docs/logo.md

```diff
@@ -0,0 +1,85 @@
+<div align="center">
+	<h1>Official BuddyPress Logo</h1>
+</div>
```

Comment:
We need to use Markdown for the documentation, I can understand it's maybe a bit frustrating as we may not be able to center paragraph or have less control about table rendering etc... But it's a requirement.

```suggestion
# Official BuddyPress Logo
```